libraries/cbor/fuzz/fuzz_targets/fuzz_target_cbor.rs

```diff
@@ -0,0 +1,18 @@
+#![no_main]
+#[macro_use]
+extern crate libfuzzer_sys;
+extern crate alloc;
+extern crate cbor;
+
+use alloc::vec::Vec;
+
+fuzz_target!(|data: &[u8]| {
+    let encoded = cbor::read(data);
+    if let Ok(value) = encoded {
+	let mut decoded = Vec::new();
+	let result = cbor::write(value, &mut decoded);
+	assert!(result);
+	assert_eq!(decoded, data);
+    };
```

Comment:
If you want, this whole function could be written:

```rust
if let Ok(value) = cbor::read(data) {
    let mut result = Vec::new();
    assert!(cbor::write(value, &mut result));
    assert_eq!(result, data);
}
```

Reply:
Thank you for the suggestion!! ^^

Reply:
No problem :-)

Also, in Rust there is no need for semi-colon `if` statements (it's only for `let` statements I think). But apparently, there is no lint about it.